src/lib/planviz/graphviz_config.hpp

```diff
@@ -0,0 +1,36 @@
+#pragma once
+
+#include <string>
+#include <unordered_map>
+
+namespace opossum {
+
+// General layout
+struct GraphvizLayout {
+  static constexpr const char* Dot = "dot";
+  static constexpr const char* Neato = "neato";
+  static constexpr const char* FDP = "fdp";
+  static constexpr const char* SFDP = "sfdp";
+  static constexpr const char* TwoPi = "twopi";
+  static constexpr const char* Circo = "circo";
+};
+
+struct GraphvizColor {
+  static constexpr const char* Transparent = "transparent";
+  static constexpr const char* White = "white";
```

Comment:
I spoke to @mrzzzrm about this beforehand and we decided that `enum`s were a cleaner way of interacting with graphviz, especially as most people will not be aware of all the config options that we are providing. So this constants stuff should just indicate which options are available for configuration. Maybe white->white is obvious but transparent might not be known as a valid option.

However, if you think it doesn't make sense to have constants here and just use strings, then I am willing to change this.

Reply:
Following that argumentation, `Green` is not an option? Since we don't want to build a full list of all configuration options, we should keep this as simple as possible. If this was something that was handed around all over the programs (like column types), it would make sense to have those types. In here, I feel like this is a bit of overkill.

I can see arguments on both sides and there probably is no right or wrong. We can leave it like this - I still had to bring it up.